includes/comfino-core.php

```diff
@@ -296,15 +297,16 @@ public static function get_widget_init_code(Comfino_Gateway $comfino_gateway): s
                 $comfino_gateway->get_option('widget_price_observer_level'),
                 $comfino_gateway->get_option('widget_price_observer_selector'),
                 Api_Client::get_widget_script_url(),
+                \Comfino_Payment_Gateway::VERSION,
             ],
             $comfino_gateway->get_option('widget_js_code')
         );
 
         return '<script>' . str_replace(
-                   ['&#039;', '&gt;', '&amp;', '&quot;', '&#34;'],
-                   ["'", '>', '&', '"', '"'],
-                   esc_html($code)
-               ) . '</script>';
+            ['&#039;', '&gt;', '&amp;', '&quot;', '&#34;'],
```

Comment:
https://www.php.net/manual/en/function.htmlspecialchars-decode.php zastanawiam się czy można tym zastąpić `str_replace`

Reply:
A sprawdzimy. Wrzucam htmlspecialchars_decode.

Reply:
Chociaż w sumie jednak nie, bo mnie interesuje odkręcenie tylko kilku wybranych encji z kodu JS wyprodukowanych przez esc_html, a użycie tej funkcji narzucili ludzie z WordPress w czasie audytu. A jak użyję tej dedykowanej funkcji to wszystko mi ona odkręci i wywołanie esc_html straci sens.